[Build Process] eslint complains about validator.js

## Issue type

Build process.

## Description

Running any of the `gulp` build commands that run `eslint` (for example, `gulp build`) produces this error:

```
src/js/components/validator.js
  10:24  error  A space is required after '['   computed-property-spacing
  10:44  error  A space is required before ']'  computed-property-spacing

✖ 2 problems (2 errors, 0 warnings)
```

--- a/src/js/components/validator.js
+++ b/src/js/components/validator.js
@@ -7,7 +7,7 @@ module.exports = function ($el) {
     validatorName,
     validatorPattern,
     $validatorCheckbox,
-    $checkList = $(data['validationElement']);
+    $checkList = $(data.validationElement);
 
   function validate () {
     for (key in data) {

